screens/Issuers/IssuersScreen.tsx

```diff
@@ -132,7 +132,7 @@ export const IssuersScreen: React.FC<
           title={t(`errors.${controller.errorMessage}.title`)}
           message={t(`errors.${controller.errorMessage}.message`)}
           goBack={goBack}
-          tryAgain={isGenericError() ? null : controller.TRY_AGAIN}
+          tryAgain={controller.TRY_AGAIN}
```

Comment:
@KiruthikaJeyashankar Please review this change

Reply:
Only for no internetConnection error we allow users to try again as per the UI wireframes. Are we having any usecase change here @vharsh ? If no this change shall be reverted

Reply:
Confirming from the wireframe in INJI-343, both generic error as well noInternetConnectionError has try again feature.

Reply:
As per the Jira description both the types of errors need to have the try-again error button, hence removed the check.